config/sidebar-learn.json

```
@@ -103,6 +103,22 @@
 					}
       ]
   },
+  {
+        "title": "Tasks and asynchronous workflow",
+        "slug": "async",
+        "routes": [
+            {
+                "source": "learn/async/asynchronous_operations.mdx",
+                "label": "Task and asynchronous operations",
```

Any reason for the singular here? Plural sounds better to me.

```suggestion
                "label": "Tasks and asynchronous operations",
```

---

My bad. It was a typo 🙏